src/form/form.tsx

```diff
@@ -400,6 +409,20 @@ class Form<MODEL extends object> extends React.Component<Props<MODEL>, State<MOD
         }, {});
     }
 
+    private getUpdatedFields(prevModel: MODEL, model: MODEL, prevField: string = ''): Array<string> {
+        return flatten(
+            Object.keys(prevModel)
+                .map((field) => {
+                    const newField = prevField ? `${prevField}.${field}` : field;
+                    if (isObject(prevModel[field])) {
+                        return [...this.getUpdatedFields(prevModel[field], model[field], newField)];
+                    }
+                    return prevModel[field] !== model[field] ? newField : [];
+                }, Infinity)
```

Comment:
What is the point of using Infinity here?

Reply:
Ah It should be removed, It's leftover from flatMap fn.